File: 2025Irrgarten/src/Irrgarten/Juego.java
package Irrgarten;
import Irrgarten.Controller.*;
import Irrgarten.UI.*;

public class Juego {
    public static void main(String[] args) {
        Game juego = new Game(2);
        TextUI textUI = new TextUI();
        Controller controlador = new Controller(juego, textUI);
        controlador.play();
    }
}

File: IrrgartenRuby/juego.rb

require_relative 'Control/controller'
require_relative 'UI/textUI'
require_relative 'game'

module Irrgarten
    juego = Irrgarten::Game.new(2)
    vista = UI::TextUI.new
    controlador = Control::Controller.new(juego,vista)
    controlador.play
   
end
